2_base_and_sensor.ipynb

```diff
@@ -256,20 +457,64 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Capture sensor information\n",
+    "In the previous section, we code the movement of the robot using our prior knowledge about the environment. In this section, we use the sensor to acquire environmental information to move the robot.\n",
```

Comment:
"In the previous section, we controlled the movement of the robot using our prior knowledge about the environment. In this section, we will use sensors to acquire environmental information to move the robot.\n",